projects/plugins/jetpack/tests/e2e/lib/env/test-setup.js

```diff
@@ -1,34 +1,5 @@
-import fs from 'fs';
-import {
-	execWpCommand,
-	getAccountCredentials,
-	provisionJetpackStartConnection,
-} from '../utils-helper';
-import { loginToWpComIfNeeded, loginToWpSite, isBlogTokenSet } from '../flows/jetpack-connect';
-import config from 'config';
-
-async function maybePreConnect() {
-	const wpComUser = 'defaultUser';
-	const mockPlanData = true;
-	const plan = 'free';
-
-	await loginToWpComIfNeeded( wpComUser, mockPlanData );
-	await loginToWpSite( mockPlanData );
-
-	if ( process.env.SKIP_CONNECT ) {
-		return;
-	}
-
-	if ( ! ( await isBlogTokenSet() ) ) {
-		const userId = getAccountCredentials( 'defaultUser' )[ 2 ];
-		return await provisionJetpackStartConnection( userId, plan );
-	}
-
-	// We are connected. Let's save the existing connection options just in case.
-	const result = await execWpCommand( 'wp option get jetpack_private_options --format=json' );
-	fs.writeFileSync( config.get( 'temp.jetpackPrivateOptions' ), result.trim() );
-}
+import { prerequisitesBuilder } from './prerequisites';
 
 beforeAll( async () => {
-	await maybePreConnect();
+	await prerequisitesBuilder().withLoggedIn( true ).withWpComLoggedIn( true ).build();
```

Comment:
Maybe we can get rid of this file altogether? Seems quite redundant at this point

Reply:
Yeah, it makes sense. I will address this in a follow up.